ci/unit_tests.bat

```diff
@@ -1,6 +1,47 @@
 @echo off
+setlocal enabledelayedexpansion
 
-setlocal
+if not exist %WORKSPACE% (
+  echo Error: env var WORKSPACE must be defined
+  exit /B 1
+)
+
+:: see if %WORKSPACE% is alread mapped to a drive
```

Comment:
I commented out 
```
 rem %RAKEPATH% test:install-core
 rem %RAKEPATH% test:core
```
and ran this a few times (in cmd)  to test this, and the drives kept incrementing every time i ran it. (it did succesfully skip c:\ however)   

I am not familiar with the voodoo here, but trying to echo `%%G` or `%%~sfH` seemed to echo the literals not what _I assume_ they are expanding to ? 
<img width="423" alt="image" src="https://user-images.githubusercontent.com/976291/30967955-55498788-a423-11e7-9816-d3f8960c0367.png">

Reply:
uh .. this is weird. I tested the "reuse" bits in isolation and it worked in my 2012 box ... but I did not test when included in the user_tests.bat. will do, thanks!

Reply:
fixed with latest set of commits
<img width="442" alt="image" src="https://user-images.githubusercontent.com/976291/30990869-7612dc68-a468-11e7-9abe-0fea2e661c12.png">